### DISCUSSION TITLE
Can't get a POST API call to work from Cypress

### DISCUSSION THREAD

**Discussion by anufrancis** [cypress-io/cypress#25858] [Questions and Help]
Is there anything wrong with this script? I am new to cypress and trying to make a POST API call from my spec file. The same API works outside using POSTMAN, but using Cypress I get 415 not supported error at cy.request. 

cy.request({
	  method: 'POST',
	  url: 'https://test-apipostcall/rest/ng/query', 
	  auth: {
		username: 'test@admin.com',
		password: '#######',
	  },
	  body: {
		"cpId": 709916,
		"aql": "select Specimen.label, Specimen.availableQty, Specimen.specimenPosition.containerName,Specimen.activityStatus where  Specimen.id=10884195"	 
	  },
		}).then(res => {
		expect(rows[0]).to.eq('fulldisspecimen');
	 });

**Comment by elTencha23** [comment]
Usually the username and password go inside the body

```
cy.request({
  method: 'POST',
  url: '/login_with_form', // baseUrl is prepend to URL
  form: true, // indicates the body should be form urlencoded and sets Content-Type: application/x-www-form-urlencoded headers
  body: {
    username: 'jane.lane',
    password: 'password123',
  },
})
```
